walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -37,7 +37,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Convert the server stats to JSON
     ServerStats serverStats = new ServerStats(startTime, currentTime, maxMemory, usedMemory);
-    String json = convertToJson(serverStats);
+    String json = convertToJsonUsingGson(serverStats);
```

Comment:
No need to commit changes when playing around with the examples. Revert this change.